waiter/src/waiter/handler.clj

```diff
@@ -176,6 +176,10 @@
                                                            (when started-at
                                                              (du/str-to-date started-at)))))]
                     (notify-instance-killed-fn instance)))
+                (log/log "InstanceTracker" :debug nil (str "Instance " instance-id " BLACKLIST " (utils/clj->json
```

Comment:
I would prefer if you declared a macro similar to `scheduler/log` that delegates to `(log/log "InstanceTracker" :debug nil ...)`.